source/_components/sensor.trafikverket_weatherstation.markdown

```diff
@@ -29,31 +30,48 @@ sensor:
     name: Trafikverket Road WeatherStation Kungälv
     api_key: eXXcbXXXacXXXXc39XX3aXXX4aXX46XX
     station: Kungälv
-    type: road
+    monitored_conditions:
+        - air_temp
+        - road_temp
+        - humidity
+        - precipitation
+        - wind_direction
+        - wind_direction_text
+        - wind_speed
 ```
 
 {% configuration %}
-name:
-  description: Unique name of the device in the frontend.
-  required: true
-  type: string
-api_key:
-  description: API key from Trafikverket.
-  required: true
-  type: string
-station:
-  description: Name of the weather station.
-  required: true
-  type: string
-type:
-  description: Defines which temperature you want (`air` or `road`).
-  required: true
+name:  
+  description: Your unique name of the device in the frontend.  
+  required: true  
   type: string
-scan_interval:
-  description: How frequently to query for new data (in seconds).
-  required: false
-  type: int
-  default: 300
+api_key:  
+  description: Your personal API key from Trafikverket.  
+  required: true  
+  type: string  
+station:  
+  description: Name of the weather station from Trafikverket.  
+  required: true  
+  type: string  
+monitored_conditions: 
+  description: Conditions to display in the frontend.  
```

Comment:
This doesn't have anything to do with the frontend per se, but decides what entities will be created for the platform, ie what states to track.

Reply:
Good input, i will clarify